.dscanner.ini

```diff
@@ -71,10 +72,48 @@ static_if_else_check="enabled"
 ; Check for unclear lambda syntax
 lambda_return_check="enabled"
 ; Check for auto function without return statement
-auto_function_check = "enabled"
-; Check for explicitly annotated unittests
-explicitly_annotated_unittests = "enabled"
+auto_function_check="enabled"
 ; Check for sortedness of imports
-imports_sortedness = "disabled"
+imports_sortedness="enabled"
+; Check for explicitly annotated unittests
+explicitly_annotated_unittests="enabled"
+; Check for properly documented public functions (Returns, Params)
+properly_documented_public_functions="enabled"
 ; Check for useless usage of the final attribute
-final_attribute_check = "enabled"
+final_attribute_check="enabled"
+; Check for virtual calls in the class constructors
+vcall_in_ctor="enabled"
+; Check for useless user defined initializers
+useless_initializer="enabled"
+; Check allman brace style
+allman_braces_check="disabled"
+; Check for redundant attributes
+redundant_attributes_check="enabled"
+
+; Configure which modules are checked with a specific checker
+; Please help to extend these checks onto more Phobos modules
+[analysis.config.ModuleFilters]
+asm_style_check="+std.algorithm"
```

Comment:
Yes `-` (blacklisting) is supported as well:
https://github.com/dlang-community/D-Scanner/blob/d9df33a53f03dd0647316c64c848a8c214bb4a0d/README.md#selecting-modules-for-a-specific-check

I am a huge fan of using blacklists as well, I was just too lazy to generate a specific module blacklist list for every non-enabled check. However the `explicitly_annotated_unittests` check is an example of using blacklists